src/daos/lib/JobsDataProviderDAO.js

```diff
@@ -212,17 +212,19 @@ export default class JobsDataProviderDAO extends AbstractContractDAO {
         flowType,
         paused,
         defaultPay,
+        startedAt,
         pausedAt,
         pausedFor,
+        finishedAt,
         requestedAdditionalTime,
         extra: new JobExtraModel({
           // TODO Fetch counts
           createdAt,
           acceptedAt,
           pendingStartAt,
-          startTime,
+          startTime: startedAt,
           pendingFinishAt,
-          finishTime,
```

Comment:
do not delete it from here
it breaks model contract which other pages depends on